ci-operator/jobs/coreos/coreos-assembler/coreos-coreos-assembler-rhel-9.6-presubmits.yaml

```
@@ -56,27 +57,30 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )images,?($|\s.*)
   - agent: kubernetes
-    always_run: true
+    always_run: false
     branches:
-    - ^2025\.4-release$
-    - ^2025\.4-release-
-    cluster: build01
-    context: ci/prow/pr-image-mirror
+    - ^rhel-9\.6$
+    - rhel-9\.6-
```

we lost the `^` here

```suggestion
    - ^rhel-9\.6-
```